list.sh

```diff
@@ -0,0 +1,6 @@
+#array=(common config connection-provider consul facade logger mdc svc-info thrift)
```

Comment:
why do we need this?

<img width="318" alt="image" src="https://user-images.githubusercontent.com/5288046/184220125-d9bf509a-cef2-44d8-a29e-b3fa802ef5de.png">